source/rdt/rdt.go

```diff
@@ -31,9 +31,10 @@ const (
 // Source implements FeatureSource.
 type Source struct{}
 
+// Name returns a friendly name for this source.
```

Comment:
Maybe change this description to `Name returns a identifier string for this feature source`?